lib/tasks/npm-install.js

```diff
@@ -27,11 +27,11 @@ class NpmInstallTask extends NpmTask {
   }
 
   formatStartMessage(packages) {
-    return `${this.useYarn ? 'Yarn' : 'NPM'}: Installing ${formatPackageList(packages)} ...`;
+    return `${this.useYarn ? 'Yarn' : 'npm'}: Installing ${formatPackageList(packages)} ...`;
```

Comment:
This seems a little out of place, like someone might come along in the future and rename to "Npm" because it matches the rest.

Reply:
As far as I understand, `Npm` is not correct either.

This particular instance is actually the one I noticed and prompted me to fix it - here:

<img width="266" alt="npm" src="https://user-images.githubusercontent.com/520420/30084229-d6fb700a-9291-11e7-9328-68ee2bfcb88a.png">

Reply:
I guess it's kind of like saying that someone might rename "iPhone" to "Iphone" because it's next to "Android".

Reply:
Good point. We will just have to be our own eslinters when looking out for this in the future!